### PR TITLE
Show all events instead of the schedule

### PR DESCRIPTION
**Pull request by Ana06**
Show all events instead od the schedule if there are less than 2 rooms. The schedule looks ugly with only one room:

image

### CODE REVIEW

File: app/controllers/schedules_controller.rb [hunk 1]
```diff
@@ -29,6 +29,9 @@ def show
   def events
     @dates = @conference.start_date..@conference.end_date
 
+    rooms = @conference.venue.try(:rooms)
+    @several_rooms = rooms && rooms.length > 1
```
Comment (ChrisBr) [inline app/controllers/schedules_controller.rb]: The temp variable does not add any value!
```
@several_rooms = @conference.venue.try(:rooms) && @conference.venue.try(:rooms) > 1
```

**Comment by ChrisBr** [comment]
Otherwise LGTM

**Comment by differentreality** [comment]
I don't agree with this change.
If people click on schedule, they expect to see a schedule (as long as it exists, eg. when we are in the middle of the conference).
Moreover we are changing the layout (hiding the schedule tab) and that's just confusing for people.

I think we should let people see the schedule as it is. If you think there is something we can do to beautify it, cool, let's do that. Otherwise people will get to see something uglier. They can always manually change to the list of events if they want to.

**Comment by Ana06** [comment]
I have been thinking about it and @differentreality is right, it would be confusing. I'll close this PR.

**Pull request closed without merging**